### DISCUSSION TITLE
Ability to leave the match during mid-fight

### DISCUSSION THREAD

**Discussion by GoodestEnglish** [diamond-rip/Eden#208] [To-do List]
Status: Not started (22/3/2023), Finished (23/5/2023)

Mentioned in #195

**Comment by HierarchyMan** [comment]
Update this

**Reply by GoodestEnglish** [reply]
Sorry but what do you mean by 'update this'?

**Reply by HierarchyMan** [reply]
You already added this so 'update this discussion and set it to completed'

**Reply by GoodestEnglish** [reply]
![image](https://github.com/diamond-rip/Eden/assets/45838505/a2d20c3e-c75c-4f40-b50e-dd809e3a13ff)
It is already display as 'finished' tho

**Reply by HierarchyMan** [reply]
Me dum nvm
